pkg/keystore/keystore.go

```diff
@@ -136,7 +145,33 @@ func (ks *Keystore) MaskTrustedKeySystemRoot(fingerprint string) (string, error)
 	return dst, ioutil.WriteFile(dst, []byte(""), 0644)
 }
 
-func (ks *Keystore) TrustedKeyPrefixExists(prefix string, r io.ReadSeeker) (bool, error) {
+// TrustKeyPrefixExists returns whether or not there exists 1 or more trusted keys for a given prefix.
+func (ks *Keystore) TrustedKeyPrefixExists(prefix string) (bool, error) {
+	acidentifier, err := types.NewACIdentifier(prefix)
+	if err != nil {
+		return false, err
+	}
+
+	pathNamesPrefix := []string{
+		// example: /etc/rkt/trustedkeys/prefix.d/coreos.com/etcd
+		path.Join(ks.LocalPrefixPath, acidentifier.String()),
+		// example: /usr/lib/rkt/trustedkeys/prefix.d/coreos.com/etcd
+		path.Join(ks.SystemPrefixPath, acidentifier.String()),
+	}
+
+	for _, p := range pathNamesPrefix {
+		_, err := os.Stat(p)
+		if err == nil {
+			return true, nil
```

Comment:
If the file exists, the key might still not be there: it could be an empty file in /etc to disable the key in /usr. Arguably, this is an existing bug, but since `TrustedKeyPrefixExists` is now used for `rkt trust`, this should be fixed...